pennylane/data/attributes/pauli.py

```diff
@@ -0,0 +1,44 @@
+import json
+from typing import Tuple, Type
+
+from pennylane.data.base.attribute import AttributeType
+from pennylane.data.base.typing_util import HDF5Array, HDF5Group
+from pennylane.pauli import PauliSentence, PauliWord
+
+
+class DatasetPauliWord(AttributeType[HDF5Array, PauliWord, PauliWord]):
+    type_id = "pauli_word"
+
+    @classmethod
+    def consumes_types(cls) -> Tuple[Type[PauliWord]]:
+        return (PauliWord,)
+
+    def hdf5_to_value(self, bind: HDF5Array) -> PauliWord:
+        return PauliWord(json.loads(bind.asstr()[()]))
```

Comment:
The brackets here remind me of a [certain programming language](https://en.wikipedia.org/wiki/Brainfuck). 😆

Reply:
yep haha, this is actually one of the things that turned me off of HDF5 at first